scripts/before_pectra_upgrade.py

```diff
@@ -0,0 +1,217 @@
+    return proxy.address, proxy.setConsensusVersion.encode_input(CS_FEE_ORACLE_CONSENSUS_VERSION)
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[TransactionReceipt]]:
+    """Prepare and run voting."""
+
+    vote_desc_items, call_script_items = zip(
+        (
+            "1. Grant MANAGE_CONSENSUS_VERSION_ROLE role on Accounting Oracle to Aragon Agent",
+            agent_forward(
+                [
+                    encode_oz_grant_role(
+                        contract=contracts.accounting_oracle,
+                        role_name="MANAGE_CONSENSUS_VERSION_ROLE",
+                        grant_to=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "2. Update Accounting Oracle consensus version to 3",
+            agent_forward([encode_ao_set_consensus_version()]),
+        ),
+        (
+            "3. Revoke MANAGE_CONSENSUS_VERSION_ROLE role on Accounting Oracle from Aragon Agent",
+            agent_forward(
+                [
+                    encode_oz_revoke_role(
+                        contract=contracts.accounting_oracle,
+                        role_name="MANAGE_CONSENSUS_VERSION_ROLE",
+                        revoke_from=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "4. Grant MANAGE_CONSENSUS_VERSION_ROLE role on Validator Exit Bus Oracle to Aragon Agent",
+            agent_forward(
+                [
+                    encode_oz_grant_role(
+                        contract=contracts.validators_exit_bus_oracle,
+                        role_name="MANAGE_CONSENSUS_VERSION_ROLE",
+                        grant_to=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "5. Update Validator Exit Bus Oracle consensus version to 3",
+            agent_forward([encode_vebo_set_consensus_version()]),
+        ),
+        (
+            "6. Revoke MANAGE_CONSENSUS_VERSION_ROLE role on Validator Exit Bus Oracle from Aragon Agent",
+            agent_forward(
+                [
+                    encode_oz_revoke_role(
+                        contract=contracts.validators_exit_bus_oracle,
+                        role_name="MANAGE_CONSENSUS_VERSION_ROLE",
+                        revoke_from=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "7. Grant MANAGE_CONSENSUS_VERSION_ROLE role on CSFeeOracle to Aragon Agent",
+            agent_forward(
+                [
+                    encode_oz_grant_role(
+                        contract=contracts.cs_fee_oracle,
+                        role_name="MANAGE_CONSENSUS_VERSION_ROLE",
+                        grant_to=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "8. Update CSFeeOracle consensus version to 3",
```

Comment:
Will it be received from `CS_FEE_ORACLE_CONSENSUS_VERSION`?
```suggestion
            "8. Update CSFeeOracle consensus version to 2",
```

Reply:
right, it should be 2. Thank you!